README.md

```diff
@@ -71,6 +70,11 @@ npm install
 npm start
 ```
 
+A word of caution:
+
+- While it is the most recent codebase, this is not production-ready!
+- This will fail if you run it as root, which is not recommended anyway.
```

Comment:
Agreed, but I would also change it to this:

```
While it is the most recent codebase, this is not production-ready!
It is not recommended to run this as root. However, if you decide to do so, you will have to run `npm run build`.
```

The "anyway" makes it sound a bit too casual.